rfcs/20180907-contrib-sunset.md

```diff
@@ -0,0 +1,280 @@
+| benchmark_tools    | saeta            | delete                               |
+| bigtable           | saeta            | move to tf.data                      |
+| boosted_trees     | nataliaponomareva | delete (moved to core)               |
+| checkpoint         | allenlavoie      | move to core                         |
+| cloud              | saeta            | merge with bigtable/                 |
+| cluster_resolver   | frankchn         | move to core                         |
+| cmake              | mrry             | delete (replaced with bazel)         |
+| coder   | nickj-google jonycgn sungjinhwang | move to tensorflow/compression |
+| compiler           | tatatodd         | move to core                         |
+| constrained_optimization | acotter    | move to separate repo                |
+| copy_graph         |                  | delete (no owner)                    |
+| crf                |                  | merge into tensorflow/probability?   |
+| cudnn_rnn          |                  | delete (available in tf.keras.layers)|
+| data               | mrry             | delete (moved to core)               |
+| decision_trees     | salehay          | delete                               |
+| deprecated         |                  | delete (deprecated)                  |
+| distribute         | josh11b priyag   | move to core                         |
+| distributions      | jvdillon     | delete (moved to tensorflow/probability) |
+| eager              | asimshankar      | move to core                         |
+| estimator          | ispirmustafa mikecase | move to tensorflow/estimator    |
+| factorization      | agarwal-ashish   | delete (rebuild in core pending)?    |
+| feature_column     | ispirmustafa     | move to core (experimental)          |
+| ffmpeg             | fredbertsch      | delete                               |
+| framework          |               | partially move to core, delete the rest |
+| fused_conv         |                  | delete                               |
+| gan                | joel-shor        | move to separate repo                |
+| gdr            |*byronyi (poxvoculi)  |                                      |
+| graph_editor       | purpledog        | delete                               |
+| grid_rnn           |(qlzh727 ebrevdo) | delete                               |
+| hadoop             | yongtang (mrry)  | move to tensorflow/io?               |
+| hooks              | ispirmustafa     | delete (already in core)             |
+| hvx                | satok16          | delete (redundant with NNAPI)        |
+| image              |                  | partial move to tensorflow/contrib?  |
+| input_pipeline     | rohan100jain     | delete                               |
+| integrate          | shoyer *mcoram   | move to tensorflow/scientific?       |
+| kafka              | yongtang (mrry)  | move to tensorflow/io?               |
```

Comment:
Lets add `ignite` ([#22210](https://github.com/tensorflow/tensorflow/pull/22210)) and `igfs` ([#22194](https://github.com/tensorflow/tensorflow/pull/22194)) here?